File: javascript/leap.js
export const isLeap = (leapYear) => {
  let leap = false;

  if (leapYear % 4 == 0) {
    leap = true;
    if (leapYear % 100 == 0) {
      leap = false;
    } 
     if (leapYear % 100 == 0 && leapYear % 400 == 0) {
      leap = true;
    }
     
  }
  else{
    leap = false;
  }

  return leap;
};

File: c#/leap.cs
using System;

public static class Leap
{
    public static bool IsLeapYear(int year)
    {
        bool leap = false;
        if (year % 4 == 0)
        {
            leap = true;
            if (year % 100 == 0 && year % 400 != 0)
            {
                leap = false;
            }
        }
        return leap;
    }
}